glide.yaml

```diff
@@ -0,0 +1,9 @@
+package: main
+import:
+- package: gopkg.in/natefinch/lumberjack.v2
```

Comment:
we should probably lock these to a known good version. I think glide 0.8.0 will resolve that issue in Masterminds/glide#143 though so use your own discretion here.